text-to-speech/v1.ts

```diff
@@ -87,11 +86,13 @@ class TextToSpeechV1 extends GeneratedTextToSpeechV1 {
       params.token_manager = this.tokenManager;
     }
 
+    // include analytics headers
+    const sdkHeaders = getSdkHeaders('text_to_speech', 'v1', 'synthesizeUsingWebSocket');
```

Comment:
Will this break the browser users?

Reply:
Hmm I'm not sure. We were still sending the `user-agent` header before but not the `x-ibm-cloud-analytics` header so it depends on if that will cause CORS issues.

That said, these two operations are the ones we have a completely separate browser package for so I don't know if it's a big deal 🤷‍♂